### PR TITLE
[IMP] - Add default value for batch_booking

### PR DESCRIPTION
**Pull request by sbejaoui**
This PR add a default value for account payment order based on account payment mode setting.

### CODE REVIEW

File: account_banking_pain_base/models/account_payment_mode.py [hunk 1]
```diff
@@ -33,3 +33,8 @@ class AccountPaymentMode(models.Model):
         "the PAIN files generated by Odoo. This value is determined by the "
         "financial institution that will process the file. If not defined, "
         "no scheme will be used.\n")
+
+    default_batch_booking = fields.Boolean(
+        string='Default Batch Booking',
+        help="Default value for payment order batch booking related "
+             "to this payment mode")
```
Comment (sbidoul) [inline account_banking_pain_base/models/account_payment_mode.py:40]: Don't you need to add the field to the view too?

Comment (sbejaoui) [reply account_banking_pain_base/models/account_payment_mode.py:40]: yes

**Comment by sbidoul** [comment]
@sbejaoui batch_booking is defined in account_banking_pain_base, so I think the default value must be defined in the same module no?

**Comment by sbejaoui** [comment]
@sbidoul , I did it in the first place, but I realized after that there was no dependency between`account_payment_order` and `account_banking_pain_base`.

https://github.com/OCA/bank-payment/blob/3c8913335067a2d3a136b50c921c8579f0010fef/account_payment_order/__manifest__.py#L20-L23

I got an attribute error in travis as you can check.

**Comment by sbidoul** [comment]
@sbejaoui account_banking_pain_base depends on account_payment_order, so that should work.

**Comment by williamphilippin-droid** [comment]
LGTM!

**Pull request closed without merging**